recipes/pyseidon/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "2.0" %}
+
+package:
+    name: pyseidon
+    version: {{ version }}
+
+source:
+    fn: v{{ version }}.tar.gz
+    url: https://github.com/GrumpyNounours/PySeidon/archive/v{{ version }}.tar.gz
+    sha256: b6c8332cfb17ec5a41ffee8c3905b60723029b3667f3e49a42fe63823265ef3f
+    patches:
+        - setup.patch
```

Comment:
I sent a patch some time ago but they insist on having this:

https://github.com/GrumpyNounours/PySeidon/blob/master/setup.py#L10